scripts/bfasst/compare_waveforms/tools/run_vivado.py

```diff
@@ -47,9 +49,9 @@ def create_tcl(template, temp_tcl, args):
                 output.write(line)
 
 
-def main():
+def launch_vivado():
```

Comment:
`launch_vivado` should take `base`, `module`, `temp`, and `vivado` as parameters to the function, and be passed those by main.  That way you can call `launch_vivado` from another script.  Currently you can't, because `args` will be `None`.